package.json

```diff
@@ -40,18 +40,19 @@
     "moment": "^2.20.1",
     "propagating-hammerjs": "^1.4.6",
     "vis-data": "^6.1.1",
-    "vis-util": "^1.1.3"
+    "vis-util": "~1.1.4"
```

Comment:
Please change this back to a caret range using `^`. We would like to auto update to minor version changes.
See https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1

Reply:
Done.